spec-main/chromium-spec.ts

```diff
@@ -367,6 +367,13 @@ describe('web security', () => {
   });
 });
 
+ifdescribe(process.platform === 'linux')('subprocesses', () => {
+  it('does not propagate GDK_BACKEND', () => {
+    const backend = process.env.GDK_BACKEND;
```

Comment:
I think we should start a child app and explicitly set its environment variable to do the test, we don't know if the test environment has the `GDK_BACKEND` set.